packages/client/hmi-client/src/components/workflow/ops/document/tera-document-operation-drilldown.vue

```diff
@@ -113,6 +113,19 @@ const docText = ref<string | null>();
 const isFetchingPDF = ref(false);
 const clonedState = ref(cloneDeep(props.node.state));
 
+function updateOutputPort() {
+	const outputPort = cloneDeep(props.node.outputs?.find((port) => port.type === 'documentId')) || null;
+	if (!outputPort) return;
+	outputPort.value = [
+		{
+			...outputPort.value?.[0],
+			equations: clonedState.value.equations
+		}
+	];
+
+	emit('append-output', outputPort);
+}
```

Comment:
- We have different meanings for "appending-output" and "update-output", please rename the function so to not cause confusion down the road.

- I do not know how operator state is managed in this operator, as usually we do not clone the state from existing output, but rather clone from the current operator's state. The idea is that if we need to switch output we would be able to restore the state that was used to derive the output. If this isn't clear, let me know and I can explain in better detail, if it is irrelevant in this operator feel free to ignore this comment.